x-pack/legacy/plugins/siem/public/components/url_state/index.tsx

```diff
@@ -42,49 +35,41 @@ UrlStateContainer.displayName = 'UrlStateContainer';
 
 const makeMapStateToProps = () => {
   const getInputsSelector = inputsSelectors.inputsSelector();
-  const getHostsFilterQueryAsKuery = hostsSelectors.hostsFilterQueryAsKuery();
-  const getNetworkFilterQueryAsKuery = networkSelectors.networkFilterQueryAsKuery();
+  const getGlobalQuerySelector = inputsSelectors.globalQuerySelector();
+  const getGlobalFiltersQuerySelector = inputsSelectors.globalFiltersQuerySelector();
+  const getGlobalSavedQuerySelector = inputsSelectors.globalSavedQuerySelector();
   const getTimelines = timelineSelectors.getTimelines();
   const mapStateToProps = (state: State, { pageName, detailName }: UrlStateContainerPropTypes) => {
     const inputState = getInputsSelector(state);
     const { linkTo: globalLinkTo, timerange: globalTimerange } = inputState.global;
     const { linkTo: timelineLinkTo, timerange: timelineTimerange } = inputState.timeline;
 
-    const page: LocationTypes | null = getCurrentLocation(pageName, detailName);
-    const kqlQueryInitialState: KqlQuery = {
-      filterQuery: null,
-      queryLocation: page,
+    const timeline = Object.entries(getTimelines(state)).reduce(
+      (obj, [timelineId, timelineObj]) => ({
+        id: timelineObj.savedObjectId != null ? timelineObj.savedObjectId : '',
+        isOpen: timelineObj.show,
+      }),
+      { id: '', isOpen: false }
+    );
```

Comment:
This looks shared with what's over in `navigation` -- may want to extract it to a re-usable helper if it has use elsewhere.

https://github.com/elastic/kibana/blob/67a180ef93832af781043f1220edce5315a5f07f/x-pack/legacy/plugins/siem/public/components/navigation/index.tsx#L101-L107